sherpa/bin/pruned_stateless_emformer_rnnt2/decode.py

```diff
@@ -15,8 +15,8 @@
 # limitations under the License.
 
 import math
-from typing import List
-
+from typing import List, Optional
+import k2
```

Comment:
We are using [pep8](https://peps.python.org/pep-0008/#imports) style here:

```
Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.

You should put a blank line between each group of imports.
```

And we use [isore](https://github.com/PyCQA/isort) to sort imports automatically

Reply:
Usually the pre-commit hook was taking care of it, that's why I notice that there are none. Will fix it

Reply:
It should be fine by now